ClassHomework/src/APCSWork/Triangle.java

```diff
@@ -0,0 +1,68 @@
+package APCSWork;
+
+import java.util.ArrayList;
+
+public class Triangle extends Polygon{
+	public Triangle() {
+		super(3, new ArrayList<Point>());
+	}
+	public Triangle(ArrayList<Point> coors) {
+		super(3, coors);
+	}
+
+	public Triangle(Point a, Point b, Point c) {
+		super(3, new ArrayList<Point>());
+		super.addOneCoordinate(a.x, a.y);
+		super.addOneCoordinate(b.x, b.y);
+		super.addOneCoordinate(c.x, c.y);
+		
+	}
```

Comment:
honestly, I don't think you need this since my design is to avoid making a constructor like this. It doesn't work if this is an octagon or with more sides.